### ISSUE TITLE
Telegram tray icon is white

### ISSUE THREAD

**Issue by oddtwelve** [telegramdesktop/tdesktop#26983]
### Steps to reproduce

1. Launch Telegram
2. Enable the tray icon
3. Observe the tray icon color, if you can find it

### Expected behaviour

The icon should be Telegram logo (blue)

### Actual behaviour

The icon is white

### Operating system

Windows 11 23H2 25982.1000

### Version of Telegram Desktop

4.11 x64

### Installation source

Static binary from official website

**Comment by oddtwelve** [comment]
![image](https://github.com/telegramdesktop/tdesktop/assets/195200/667f20bc-f04d-4dc3-89fc-76db21595c10)

Possible duplicate of #7566

**Comment by john-preston** [comment]
@oddtwelve Thats intended.

**Comment by john-preston** [comment]
This was a move to be consistent with system icons, they're monochrome in modern apps. Related issue https://github.com/telegramdesktop/tdesktop/issues/17438

**Comment by ildar170975** [comment]
The latest change made the icon UNRECOGNIZABLE
![изображение](https://github.com/telegramdesktop/tdesktop/assets/71872483/7e905a12-409e-4c03-b1bf-7dcd8d822415)

Please be more reasonable and do not follow "modern trends" blindly.

**Comment by 29988122** [comment]
Please at least give us an option to change it back : (

Thanks..!

**Comment by Zeeex** [comment]
So disregard accessibility only to conform to the Windows/Flat "Modern" UI? Shame 😢

Like @29988122 said, when you make a change like this, it's common decency to provide a way to change back, at least.

As for me, it's time to perma downgrade to 4.10.2, unless this gets fixed.

**Comment by Aokromes** [comment]
> As for me, it's time to perma downgrade to 4.10.2, unless this gets fixed.

not a bug it will not be "fixed" because ain't broken.

**Comment by lazynooblet** [comment]
I came here to say there was no system tray icon, only to find this and go "oh! its just completely hidden now"

> This was a move to be consistent with system icons

Telegram isn't a system-app, its allowed to have colours, like all the other apps that are not system apps.

**Comment by Mike-Mercer** [comment]
Its not broken, its just ugly...

**Comment by QuAzI** [comment]
It’s amazing to read in the modern world of noise and distracting glitter that for someone distraction-free icon is such a problem

**Comment by dajotim937** [comment]
Jesus. Is it a company politic to make UI change to worse because of modern trend or what?
First rounded corners, now shitty absolutely indistinguishable and invisible white icon.

Great job. /s

**Comment by Artyom-K** [comment]
For some reason with this new icon I'm not getting unread count when I'm mentioned in muted group chats. I always had "Include muted chats in unread count" disabled and it was very convenient to see that someone mentioned me and I haven't read it. Now it doesn't work that way. It still works as it was for unmuted chats.

**Comment by vladadmin** [comment]
Please add the ability for the user to select the type of icon, for example with check marks: one check mark is the old look - blue, the second check mark - makes a new worthless look).
Now, instead of a split second, I spend 2-5 seconds searching for the tray icon among other icons to get information about my messages. If earlier a quick glance at the icon in the tray was enough for me, now I need to stop work, look for the icon, and return to work. It's very distracting and really gets in the way.
Give the user a choice - this will be the best solution.

**Comment by Zeeex** [comment]
> not a bug it will not be "fixed" because ain't broken.

I didn't say it was a bug, but an accessibility regression. That's could be fixable with colors, or a better brand identity, which current icon doesn't provide.

Not against flat UI, it sometimes looks good, but there are some exceptions.

**Comment by Emil-babayan** [comment]
Literally the worst icon I've seen in my 36 years of living,

**Comment by Codyport** [comment]
Clearly for the sake of consistency, all icons should be round, the identical shade of grey-on-grey, and indistinguishable from one another.  Colour and design serves a purpose and I'm disappointed to see Telegram make such a poor UX choice.  There is a reason traffic lights aren't just three white lights, there's a reason ambulances aren't the same colour as moving vans etc.  IMHO this is a poor design choice - it's not just change-shock we're feeling, we're genuinely incredulous.

**Comment by catterpiler74** [comment]
вот ребят, нашел человечка кто считает что белая иконка лучше, себе на лбу нарисуй иконку белую @ilya-fedin д*****еб

**Comment by catterpiler74** [comment]
https://telegram-for-desktop.ru.uptodown.com/windows/post-download/109082955
old version without white shit icon

**Comment by lazynooblet** [comment]
> https://telegram-for-desktop.ru.uptodown.com/windows/post-download/109082955 old version without white shit icon

Please don't download from random sites. This github hosts old versions, there is no need to put yourself at risk.

**Comment by catterpiler74** [comment]
ну так оставь ссылку д****еб на версию 4.10.2 с github и выкинь на главную а че ты тут умничаешь сделай все до конца. я в шоке вас говном мажут а вы радостные попиваете латте на соевом молоке.... кто понял тот понял , "это не наши"

**Comment by r10v** [comment]
Win Task manager tray icon is green. Bluetooth system icon is blue and white. Nvidia icon is green and black. But Telegram... Well I guess everything else is added and fixed so only issue was blue icon 🤦‍♂️

**Comment by Aokromes** [comment]
stop spamming issue, this already changed, there is no need of complain anymore.

**Comment by lazynooblet** [comment]
> stop spamming issue, this already changed, there is no need of complain anymore.

Did it get reverted? I don't see it.

**Comment by evidenzas** [comment]
as for me, an even more nasty problem is not that the icon is white, but that it is not recognizable due to the counter of unread messages, including for some reason the counter of messages from muted chats. for why you did that? you could just add a dot if there are unread messages, or just give the user a choice as suggested above
![изображение](https://github.com/telegramdesktop/tdesktop/assets/54247901/936e9d9b-2563-47f4-bad0-0fb30e2b7b11)

**Comment by Uasissualij** [comment]
Simply removing one registry setting is enough to return the blue icon ))
`REG DELETE HKCU\SOFTWARE\Microsoft\Windows\CurrentVersion\Themes\Personalize /v "SystemUsesLightTheme" /f
pause`

**Comment by oddtwelve** [comment]
> > stop spamming issue, this already changed, there is no need of complain anymore.
> 
> Did it get reverted? I don't see it.
> 
> Edit: I'm blind, I see the commit. Muchos gracious.

Now it has a setting for it in 4.11.2. Yay!

**Comment by Codyport** [comment]
Thanks, I wish ... well virtually any other company... was as responsive to this sort of thing!

**Comment by Codyport** [comment]
Unfortunately, confirmed, we're still now stuck with the white icon on Linux, may we have the same option to regress to the previous blue one please?  It is genuinely much easier to see when there's a message on that one.

**Comment by ilya-fedin** [comment]
In case of Linux it's decided not by Telegram itself but by your system settings (chosen icon theme)

**Comment by Codyport** [comment]
> In case of Linux it's decided not by Telegram itself but by your system settings (chosen icon theme)

It changed at the same time as my Windows machine, and there's no system setting/ icon theme to change it back - I think you're wrong here, thrilled to be shown otherwise tho?!  Maybe I can find an old version here on Github to check with.

**Comment by ilya-fedin** [comment]
AFAIK there were no changes to Linux tray icon except of support for HiDPI in tray counter (the biggest tray icon is enlarged from 48px to 256px) and [a PR from some GNOME person](https://github.com/telegramdesktop/tdesktop/pull/27035), this change doesn't have support for official builds so unless you have a distro build installed it shouldn't affect you...

**Comment by Codyport** [comment]
I don't know.  I don't get why / how the Linux install would define the icons that an arbitrary application would use.  How can the app not install its own icons?  Why would a theme have icons for Telegram?  Etc.  Anyway I will play around more when I get time, spin up a VM and mess about with it.  I can't let things like this go, either I can learn something... or I'm right haha!  thanks =)

**Comment by ilya-fedin** [comment]
Well, in Linux system icon theme can theme icons of any application, it's just a folder with icons that can override icons in any application, it's enough to just name icon in the same way as the name of icons application itself provides. I know at least KDE provides monochrome tray icons for Telegram.

**Comment by Codyport** [comment]
I killed all the Telegram install related everything, reinstalled it from Snap, and now there's a telegram.png file in my icons directory that wasn't there before and I've got the right icon back.  Weirdly this is on 2 machines - both upgraded to Kubunu 23.10 last week so I suspect it's messed me around somehow.  thanks for the pointer, if this helps someone else out somehow, it was all worth it! Now I feel like I can tell that there's messages easier.  Though... it could all be in my head =)
